Resources/Prototypes/DeltaV/Entities/Markers/Spawners/jobs.yml

```diff
@@ -13,3 +13,46 @@
       state: medical
     - sprite: Mobs/Silicon/chassis.rsi
       state: medical_e
+
+- type: entity
+  id: SpawnPointChiefJustice
+  parent: SpawnPointJobBase
+  name: chief justice
+  components:
+  - type: SpawnPoint
+    job_id: ChiefJustice
+  - type: Sprite
+    layers:
+      - state: green
+      - state: lawyer
+      #- sprite: DeltaV/Markers/jobs.rsi
+      #  state: chiefjustice
+
+- type: entity
+  id: SpawnPointClerk
+  parent: SpawnPointJobBase
+  name: clerk
+  components:
+  - type: SpawnPoint
+    job_id: Clerk
+  - type: Sprite
+    layers:
+    - state: green
+    - state: lawyer
+    #- sprite: DeltaV/Markers/jobs.rsi
+    #  state: clerk
+
+- type: entity
+  id: SpawnPointProsecutor
+  parent: SpawnPointJobBase
+  name: prosecutor
+  components:
+  - type: SpawnPoint
+    job_id: Prosecutor
+  - type: Sprite
+    layers:
+    - state: green
+    - state: lawyer
+    #- sprite: DeltaV/Markers/jobs.rsi
```

Comment:
Do we have this yet?

Reply:
Actually I think I remember saying I would make these...uh...monday, unless someone beats me to it